There is a checkstyle error:
```
ant:checkstyle] [ERROR] /home/jenkins/jenkins-slave/workspace/kafka-pr-jdk8-scala2.11@2/streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java:112:1: File contains tab characters (this is the first instance). [FileTabCharacter]
```

--- a/streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java
+++ b/streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java
@@ -109,7 +109,6 @@ public class StoreChangelogReader implements ChangelogReader {
 
                 final StateRestorer restorer = stateRestorers.get(partition);
                 restorer.setCheckpointOffset(StateRestorer.NO_CHECKPOINT);
-				
                 task.reinitializeStateStoresForPartitions(recoverableException.partitions());
             }
             restoreConsumer.seekToBeginning(partitions);

